internal/sdkv2provider/resource_cloudflare_workers_script.go

```diff
@@ -131,6 +131,12 @@ func parseWorkerBindings(d *schema.ResourceData, bindings ScriptBindings) {
 	}
 }
 
+func getCompatibilityFlags(d *schema.ResourceData, compatibilityFlags []string) {
+	for _, item := range d.Get("compatibility_flags").([]interface{}) {
+		compatibilityFlags = append(compatibilityFlags, item.(string))
+	}
+}
+
```

Comment:
if you're pulling the values from the schema in this method, you don't need to pass in the slice of strings. instead, you can contain it all in the method and reference that for your structs.

```suggestion
func getCompatibilityFlags(d *schema.ResourceData) []string {
	compatibilityFlags := make([]string, 0)
	for _, item := range d.Get("compatibility_flags").([]interface{}) {
		compatibilityFlags = append(compatibilityFlags, item.(string))
	}
	return compatibilityFlags
}
```